dali/operators/generic/shapes.cc

```diff
@@ -17,7 +17,10 @@
 namespace dali {
 
 DALI_SCHEMA(Shapes)
-    .DocStr(R"code(Returns the shapes of inputs.)code")
+    .DocStr(R"(Returns the shapes of tensors in the input batch.
+
+NOT RECOMMENDED FOR NEW CODE. Use `DataNode.shape()` instead.
+)")
```

Comment:
Do we have a way do deprecate it only in documentation and not scream when it's used? Because `DataNode.shape` uses `fn.shapes` internally. It's just not recommended for direct use. Perhaps we can add an alias and use the alias instead.

Reply:
Done.